### DISCUSSION TITLE
Why there is a timestamptz option, if you can not select the timezone?

### DISCUSSION THREAD

**Discussion by mbledkowski** [supabase/supabase#1635] [Questions] [answered]
Why there is an option for a timestamp**tz** variable, if you can not change the default timezone?

**Comment by AndrewHows** [comment]
timestamp just stores a particular point in time
timestamptz stores a particular point in time **and** timezone information

timestamp: 1/Jan/2020 15:32:14
timestamptz: 1/Jan/2020 15:32:14 GMT -4

They're used for different things, regardless of the default timezone of the server.

**Comment by steve-chavez** [comment, marked as answer]
Hey @mbledkowski,

You can change the timezone from the SQL Editor, with

```sql
alter database postgres set timezone to 'Asia/Singapore';
```

You can then confirm with

```sql
show timezone;
-- Asia/Singapore
```

Also, you can see a list of the supported timezones with this query

```sql

select * from pg_timezone_names; 
```

---

Related to a previous issue: https://github.com/supabase/supabase/issues/176#issuecomment-742181889
